src/platform/packages/shared/kbn-scout/src/servers/configs/default/serverless/oblt_logs_essentials.serverless.config.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { defaultConfig } from './serverless.base.config';
+import type { ScoutServerConfig } from '../../../../types';
+
+export const servers: ScoutServerConfig = {
```

Comment:
I think it is ok to backport it to 9.1 and avoid conflicts in the future. It won't be run, but at least persist in branch